alf/examples/ddpg_pendulum.gin

```diff
@@ -27,22 +26,20 @@ DdpgAlgorithm.critic_optimizer=@critic/Adam()
 DdpgAlgorithm.target_update_period=5
 OneStepTDLoss.td_error_loss_fn=@losses.element_wise_huber_loss
 
-Agent.rl_algorithm_cls=@DdpgAlgorithm
 
 # training config
-TrainerConfig.trainer=@sync_off_policy_trainer
 TrainerConfig.initial_collect_steps=1000
-TrainerConfig.num_updates_per_train_step=1
 TrainerConfig.mini_batch_length=2
 TrainerConfig.unroll_length=1
 TrainerConfig.mini_batch_size=64
+TrainerConfig.num_updates_per_train_step=1
+TrainerConfig.whole_replay_buffer_training=False
 TrainerConfig.clear_replay_buffer=False
-TrainerConfig.algorithm_ctor=@Agent
-TrainerConfig.checkpoint_interval=100000
+TrainerConfig.algorithm_ctor=@DdpgAlgorithm
 TrainerConfig.num_iterations=20000
-TrainerConfig.evaluate=1
-TrainerConfig.eval_interval=1000
-TrainerConfig.debug_summaries=0
+TrainerConfig.num_checkpoints=5
```

Comment:
Let's make a ddpg.gin to set default arguments for DDPG and people can include it to set the right config.

Reply:
Created a ddpg.gin for sharing and importing by other ddpg-based applications.